KianCommons/UI/Table/UITable.cs

```diff
@@ -0,0 +1,83 @@
+using ColossalFramework.UI;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace KianCommons.UI.Table {
+    class UITable : UIPanel {
+        public UITableRow[] rows = new UITableRow[0];
+        private float[] columnWidths = new float[0];
+        public int RowCount {
+            get => rows.Length;
+        }
```

Comment:
[minor comment]
you can also write
public int RowCount => rows.Length

Reply:
done